### ISSUE TITLE
Is the recommendation to use metadata pytest.integrator for local versions outdated?  

### ISSUE THREAD

**Issue by schaetzc** [pypa/packaging.python.org#1503]
The last section in https://packaging.python.org/en/latest/specifications/version-specifiers/#local-version-identifiers says

> **Local version identifiers**
> [...]
> Source distributions using a local version identifier SHOULD provide the `python.integrator` extension metadata (as defined in [PEP 459](https://peps.python.org/pep-0459/)). 

However, I couldn't find any (still relevant) instructions for `python.integrator`. [PEP 459](https://peps.python.org/pep-0459/) says

> This PEP depends on [PEP 426](https://peps.python.org/pep-0426/), which has itself been withdrawn. See the PEP Withdrawal section in that PEP for details.

> In the meantime, metadata extensions will continue to be handled as they have been for past

Does that mean, `python.integrator` should not be used, and the hint in [version specifiers](https://packaging.python.org/en/latest/specifications/version-specifiers/#local-version-identifiers) is outdated and should be removed?

If yes, similar references to metadata should be removed in other documents too. E.g. https://packaging.python.org/en/latest/specifications/core-metadata/ refers to [PEP 566](https://peps.python.org/pep-0566/) as "*this is not **yet** used as a standard*", while [PEP 566](https://peps.python.org/pep-0566/) itself says

> This PEP is a historical document. The up-to-date, canonical spec, [Core metadata specifications](https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata), is maintained on the [PyPA specs page](https://packaging.python.org/en/latest/specifications/).

This looks like cycle of outdated documents that refer to each other as being the future. Or am I missing something?

**Comment by chrysle** [comment]
> Does that mean, `python.integrator` should not be used, and the hint in [version specifiers](https://packaging.python.org/en/latest/specifications/version-specifiers/#local-version-identifiers) is outdated and should be removed?

Looks like that should be done, yes.

> This looks like a cycle of outdated documents that refer to each other as being the future. Or am I missing something?

Normally, if a packaging PEP is considered stable and is transferred to PyPUG (this guide), it will be marked as _"historical"_ on the Python PEPs page and maintenance is handed over to the guide editors.

**Comment by webknjaz** [comment]
cc @ncoghlan looks like that's your PEP mentioned ^

**Comment by ncoghlan** [comment]
Yep, this is a straight up editing oversight in PEP 440 which then carried over to the packaging user guide. We can just delete the offending paragraph.
